app/components/task/details/output/task-outputs.component.ts

```diff
@@ -85,7 +85,7 @@ export class TaskOutputsComponent implements OnChanges, OnDestroy {
             });
             nodeNavigator.init();
 
-            const taskOutputPrefix = `${this.task.id}/`;
+            const taskOutputPrefix = `${this.task.id}`;
```

Comment:
I think we should make the command support having a basePath ending or not with / instead of joining / and then breaking that.

Reply:
agree though i don't think there is any need to have a / here either. given that at the moment the prefix is only used by blob storage queries and the prefix has never ended in a / before. So i suggest we remove it here and also make the command handle having a path segment ending in a /